ecosystem/indexer-grpc/indexer-grpc-table-info/src/internal_indexer_db_service.rs

```diff
@@ -158,6 +158,25 @@ impl InternalIndexerDBService {
             start_version = next_version;
         }
     }
+
+    // For internal testing
+    pub async fn run_with_end_version(
```

Comment:
nit: add #cfg[test]?

Reply:
thought of this. However, api tests are not using this flag to create test context. changing those tests is probably beyond this PR.